src/components/SignupForm/index.js

```diff
@@ -1,57 +1,77 @@
 import { css } from '@emotion/core';
-import { Button, TextField } from '@octopusthink/nautilus';
+import { Paragraph, TextField } from '@octopusthink/nautilus';
 import React from 'react';
 
+import ButtonStyled from 'components/ButtonStyled';
+
 const SignupForm = () => {
   return (
-    <form
-      action="https://buttondown.email/api/emails/embed-subscribe/octolabs"
-      method="post"
-      target="popupwindow"
-      onsubmit="window.open('https://buttondown.email/octolabs', 'popupwindow')"
-      css={css`
-        text-align: center;
-
-        @media screen and (min-width: 560px) {
-          display: flex;
-          align-items: flex-end;
-        }
-      `}
-    >
-      <input type="hidden" value="1" name="embed"></input>
-      <input type="hidden" name="tag" value="turnip-beta"></input>
-      <div
+    <React.Fragment>
+      <form
+        action="https://buttondown.email/api/emails/embed-subscribe/octolabs"
+        method="post"
+        target="popupwindow"
+        onSubmit="window.open('https://buttondown.email/octolabs', 'popupwindow')"
```

Comment:
😱

I'll tweak this before we ship this 😅

Reply:
Technically I'm not sure it does anything—we might be able to remove it. 

For reference, how we're doing it elsewhere...

https://github.com/octopusthink/octopusthink.com/blob/master/src/components/NewsletterSignupForm/index.js
https://github.com/octopusthink/getmicdrop.com/blob/master/src/components/NewsletterSignupForm/index.js

...should I just copy the approach used there?